main.py

```diff
@@ -337,6 +347,8 @@ def disable_lldp(self):
             else:
                 error_list.append(id_)
         if not error_list:
+
+            self.notify_lldp_change('disabled', changed_interfaces)
```

Comment:
This must be inside a new `if changed_interfaces`, not inside `if not error_list`